Papers/InProgress/HajekAndersonControversy/main.tex

```diff
@@ -302,24 +307,22 @@
 \ref{fig:Hajek1}), which replaces Anderson's A:A1 and A2 by a simpler
 axiom H:A12. Surprisingly, the computer-assisted formalization of
 \AOEH\ shows that A4 and A5' are still superfluous. Moreover,
-A4 and A5' are independent\footnote{
+A4 and A5' are independent
+of the other axioms and definitions.
+Therefore, A4 and A5' are not redundant, despite their superfluousness.
+
+\begin{definition}
   An axiom $A$ is \emph{independent} of a set of axioms $S$ iff
   there are models of $S$ where $A$ is true and other models of
   $S$ where $A$ if false.
-}
-of the other axioms and definitions.
-Therefore, A4 and A5' are not redundant, despite their superfluousness.
+\end{definition}
 
 \begin{figure}[t]
 \noindent \framebox[\columnwidth][r]{
 \begin{minipage}{.94\columnwidth}\small
 \begin{itemize}
-\item[H:A12] The negation of a property necessarily implied by a
-  positive property is not positive:
-  $$\hol{\allq \varphi \allq \psi [(P(\varphi) \wedge \nec \allq x [\varphi(x)
-  \imp \psi(x)]) \imp \neg P(\neg \psi)]}$$
-\item[H:D1] A \emph{God-like} being necessarily possesses those and only those properties that are necessarily implied by a positive property:
-  $$\hol{G_H(x) \equiv \forall \varphi [\nec \varphi(x) \biimp \exq \psi[P(\psi) \wedge \nec \allq x [\psi(x)  \imp \varphi(x))]]}$$
+\item[H:DP] A \emph{God-like} being necessarily possesses those and only those properties that are necessarily implied by a positive property:
+  $$\hol{P(x) \equiv \forall \varphi [\nec \varphi(x) \biimp \exq \psi[P(\psi) \wedge \nec \allq x [\psi(x)  \imp \varphi(x))]]}$$
```

Comment:
I do not understand this change (replacement of H:A12 and H:D1 by H:DP). I do not remember seeing this H:DP before. Moreover, H:A12 and H:D1 seem correct according to AOE' in section 4 of Hajek 2002 and in the Isabelle file.  Why did you do this replacement, Leon?  I will revert it for now.